cinema/models.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=100)
+    description = models.CharField(max_length=255)
```

Comment:
The `description` field is defined as a `CharField` with a maximum length of 255. If the description is expected to be longer, consider using `TextField` instead, which does not have a fixed maximum length.